usersService/src/main/java/il/ac/afeka/usersservice/boundaries/DepartmentBoundary.java

```diff
@@ -44,21 +44,12 @@ public void setCreationDate(String creationDate) {
         this.creationDate = creationDate;
     }
 
-    public static String changeDateFormat(String inputDate) {
-        // Parse the input date string to a LocalDate object
-        LocalDate localDate = LocalDate.parse(inputDate, DateTimeFormatter.ISO_DATE);
-
-        // Format the LocalDate object to the desired output format
-        String outputDate = localDate.format(DateTimeFormatter.ofPattern("dd-MM-yyyy"));
-
-        return outputDate;
-    }
 
     public DepartmentEntity toEntity() {
         DepartmentEntity entity = new DepartmentEntity();
         entity.setDeptId(this.getDeptId());
         entity.setDepartmentName(this.getDepartmentName());
-        entity.setCreationDate(changeDateFormat(LocalDate.now().toString()));
+        entity.setCreationDate(DateValidator.changeDateFormat(LocalDate.now().toString()));
```

Comment:
Why passing the date as a string only to parse it into date again?
you can pass it to the function as a date and getting it back as a string